docs/content/applying-policies/quota-scheduler/quota-scheduler

```diff
@@ -0,0 +1,5 @@
+Welcome to this tutorial, which delves into a selection of strategies designed to boost the resilience and adaptability of service infrastructure. In the challenging field of service provision, it's recognized that achieving 100% service availability at all times is a lofty goal. Interruptions can be caused by a range of factors such as unpredictable client behaviors, sudden surges in demand, or service outages due to software bugs.
+
+There are scenarios where traditional methods, like distributed circuit breaking, may not be adequate despite being generally effective in controlling throughput. For example, a situation could arise when a large number of hosts are forwarding traffic to a few servers, potentially overwhelming the target hosts. Balancing these traffic patterns to prevent cascading failure can be a challenging task. In these cases, global rate limiting serves as a practical solution.
```

Comment:
Though pr is about quota scheduler, the text is talking about global API rate limiting. So I'm confused here.